src/EPR.PRN.Backend.API.UnitTests/Controllers/RegistrationControllerTests.cs

```diff
@@ -49,4 +53,117 @@ public async Task UpdateRegistrationSiteAddress_ValidCommand_ReturnsNoContent()
             command.RegistrationId.Should().Be(registrationId);
         }
     }
+
+    
+    [TestMethod]
+    public async Task GetRegistrationsOverviewForOrgId_NoRegistrations_ReturnsOkWithEmptyList()
+    {
+        // Arrange
+        var organisationId = Guid.NewGuid();
+        var query = new GetRegistrationsOverviewByOrgIdQuery { OrganisationId = organisationId };
+        var expectedResult = new List<RegistrationOverviewDto>();
+        _validationServiceMock
+            .Setup(v => v.ValidateAndThrowAsync(query, It.IsAny<CancellationToken>()))
+            .Returns(Task.CompletedTask);
+        _mediatorMock
+            .Setup(m => m.Send(query, It.IsAny<CancellationToken>()))
+            .ReturnsAsync(expectedResult);
+        // Act
+        var result = await _controller.GetRegistrationsOverviewForOrgId(organisationId);
+        // Assert
+        using (new AssertionScope())
+        {
+            result.Should().BeOfType<OkObjectResult>();
+            var okResult = result as OkObjectResult;
+            okResult!.Value.Should().BeEquivalentTo(expectedResult);
+        }
+    }
+
+    [TestMethod]
+    public async Task GetRegistrationsOverviewForOrgId_ValidOrganisationId_ReturnsOkResult()
+    {
+        // Arrange
+        var organisationId = Guid.NewGuid();
+        var query = new GetRegistrationsOverviewByOrgIdQuery { OrganisationId = organisationId };
+        var expectedResult = new List<RegistrationOverviewDto>
+        {
+            new RegistrationOverviewDto { RegistrationId = Guid.NewGuid() },
+            new RegistrationOverviewDto { RegistrationId = Guid.NewGuid() }
+        };
+        _validationServiceMock
+            .Setup(v => v.ValidateAndThrowAsync(It.IsAny<GetRegistrationsOverviewByOrgIdQuery>(), It.IsAny<CancellationToken>()))
+            .Returns(Task.CompletedTask);
+        _mediatorMock
+            .Setup(m => m.Send(It.IsAny<GetRegistrationsOverviewByOrgIdQuery>(), It.IsAny<CancellationToken>()))
+            .ReturnsAsync(expectedResult);
+        // Act
+        var result = await _controller.GetRegistrationsOverviewForOrgId(organisationId);
+        // Assert
+        using (new AssertionScope())
+        {
+            result.Should().BeOfType<OkObjectResult>();
+            var okResult = result as OkObjectResult;
+            okResult!.Value.Should().BeEquivalentTo(expectedResult);
+        }
+    }
+
+    [TestMethod]
+    public async Task GetRegistrationsOverviewForOrgId_NoRegistrations_ReturnsOkWithEmpty()
```

Comment:
This looks duplicate with GetRegistrationsOverviewForOrgId_NoRegistrations_ReturnsOkWithEmptyList